pkg/kubelet/config/common.go

```diff
@@ -132,6 +138,9 @@ func tryDecodeSinglePod(data []byte, defaultFn defaultFunc) (parsed bool, pod *v
 	if err = defaultFn(newPod); err != nil {
 		return true, pod, err
 	}
+	if err = validatePodFeatures(newPod); err != nil {
```

Comment:
this is only used for static pods / pods from a url manifest, not pods coming from the API server ... I don't think this is where you want to add this